<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(36, 159, 156)</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="level_main_menu">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>210</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;asdafadfadf&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 122, 118);
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>LEVEL 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_main_menu">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>460</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 122, 118);
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>WELCOME USER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="play_button_main_menu">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(237, 27, 123);
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:10px;
}
QPushButton:hover{
background-color: rgb(255, 255, 255);
color: #f44786;
border-color: #f44786;
border-width:3px;
}</string>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_button_main_menu">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>440</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(237, 27, 123);
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:10px;
}
QPushButton:hover{
background-color: rgb(255, 255, 255);
color: #f44786;
border-color: #f44786;
border-width:3px;
}</string>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_main_menu">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>81</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:10px;
border-color: rgb(3, 122, 118);
color: #037a76;</string>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="set_timer_main_menu">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>690</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; vertical-align:super;&quot;&gt;asdafadfadf&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(3, 122, 118);
color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
border-style: solid;
border-width:2px;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>SET TIMER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="time_button_main_menu">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>695</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
background-color: rgb(237, 27, 118);
border-radius:18px;
color: rgb(255, 255, 255);
}
QPushButton:hover{
background-color: rgb(255, 255, 255);
color: #f44786;
border-color: #f44786;
border-width:3px;
}</string>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
